### ISSUE TITLE
How to poll for Favs?

### ISSUE THREAD

**Issue by perklun** [codepath-nomadic/nomadic#36]
Hey Jean,

Did we discuss how to track favs? How are tracking the favs stored by a user?

**Comment by aurambaj** [comment]
no. Let me try to come up with some method in the client for monday
